### ISSUE TITLE
CSP errors in AAQ question thread

### ISSUE THREAD

**Issue by mirunacurtean** [mozilla/sumo#1073]
**Describe the bug**
In the Console, various inline style CSP related errros have been received on question threads. 
Example: https://support.allizom.org/en-US/questions/1207256

**To Reproduce**
1. Go to https://support.allizom.org/en-US/questions/1207256
2. Inspect page, select Console and reload.
4. See errors

**Expected behavior**
Should not log anymore CSP related errors. 

**Screenshots**
![image](https://user-images.githubusercontent.com/53306802/181236226-e084a487-d1ff-46ba-9894-1edc88997274.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows]
 - Browser [e.g. Edge, Firefox]
 - Version [102.0.1]

**Additional context**

**Comment by akatsoulas** [comment]
@smithellis care to take a stub at this?

**Comment by smithellis** [comment]
@akatsoulas - I'd like another opinion, but here is where I've landed.  There is an issue with the version of jquery we use (1.11.3) that causes FF to complain about a CSP violation. I found this blurb 

"Mozilla Firefox blocks scripts and event handlers prohibited by CSP rules even at the stage of page rendering and immediately sends a blocking report. Blocking inline event handlers in Firefox browsers occurs [before the actual occurrence of the event](https://csplite.com/csp/test1/#ff_chrome_feature_inline_event_handler), for example, clicking a button or pressing a key."

When the js block 
```
if ( !(support[ i + "Bubbles" ] = eventName in window) ) {
			// Beware of CSP restrictions (https://developer.mozilla.org/en/Security/CSP)
			div.setAttribute( eventName, "t" );
			support[ i + "Bubbles" ] = div.attributes[ eventName ].expando === false;
		}
```
is loaded, the CSP violation triggers.  The setAttribute is the cause. Allowing "unsafe-inline" or various nonce strategies might neutralize the notification, but would diminish the usefulness of having the CSP policy in place. I think the best course of action is to add a jquery upgrade to the current library upgrade efforts.
I'm open to other ideas. I tried to put in a polyfill to consume this particular event, but then thought better of it - it would be a one off fix that doesn't solve the core issue.
Thanks for any feedback.
